lesson01/assignment1/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Assignment01 Part 2 ES6</title>
+     <!-- adding JQuery -->
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
+    <!--adding javascript -->
+    <script src="ES6Methods.js"></script>
+    <!-- CSS styling -->
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+  </head>
+  <body>
+    <h1 class="padding">ES6+ Javascript functionality</h1>
+    <section class="padding border">
+      <p>Is your number greater than any in this hidden array? Enter a number to find out.</p>
+      <!-- Function input -->
+      <input class="input" type="text">
+      <!-- Sumbit input to Javascript -->
+      <input class="submit" type="submit">
+      <!-- Result shown -->
+      <p>Numbers from array greater than your number: <span class="sum"></span></p>
+    </section>
+    <section class="padding border">
+      <p>Try and figure out what the secret word is. Enter a word or a combination of letters to find out.</p>
+      <!-- Function input -->
+      <input class="inputtwo" type="text">
+      <!-- Sumbit input to Javascript -->
+      <input class="submittwo" type="submit">
+      <!-- Result shown -->
+      <p>Does your entry above match the secret word? <span class="sumtwo"></span></p>
+    </section>
+    <section class="padding border">
+      <p>Entering a number will divide the numbers in this array. Can you figure out what numbers are in this array?</p>
+      <!-- Function input -->
+      <input class="inputthree" type="text">
+      <!-- Sumbit input to Javascript -->
+      <input class="submitthree" type="submit">
+      <!-- Result shown -->
+      <p>Numbers in array divided by your number: <span class="sumthree"></span></p>
+    </section>
+    <section class="padding border">
+      <p>Enter your name here for a secret message.</p>
+      <!-- Function input -->
+      <input class="inputfour" type="text">
+      <!-- Sumbit input to Javascript -->
+      <input class="submitfour"type="submit">
+      <!-- Result shown -->
+      <p><span class="sumfour"></span></p>
+    </section>
```

Comment:
More semantic class names for the elements would make this more maintainable. For example, the final one could have names like `inputgreeting`, `submitgreeting`, etc.